server/src/main/java/com/epam/aidial/core/server/data/ResourceTypes.java

```diff
@@ -1,6 +1,6 @@
 package com.epam.aidial.core.server.data;
 
-import com.epam.aidial.core.server.resource.ResourceType;
+import com.epam.aidial.core.resourceservice.resource.ResourceType;
```

Comment:
A weird package name? com.epam.aidial.core.resource.service.ResourceType?

Reply:
right the package name `resourceservice` is a bit weird but
if you suggest `resource.service`
The rest packages will be `resource.service.resource` or `resource.service.service`

Reply:
let's rename to `storage`